tests/integration/templates/cdk_bootstrap_sequence.json

```diff
@@ -0,0 +1,27 @@
+[
```

Comment:
Pretty crude approach here (recorded API calls from `cdklocal` runs). An alternative would be to install `cdklocal` into the test environment, but that seems a bit bloated. Over time, we could also cover this functionality as part of the tests in the `https://github.com/localstack/aws-cdk-local` repo itself..

Reply:
Good point! Moved it to the `tests/integration/files/` folder, which is a better place for it. 👍  Preferring to keep it outside the test file for now, as it contains some very long lines with several 1000 characters.